lib/poller.js

```diff
@@ -37,10 +37,10 @@ class Poller extends EventEmitter {
     await this.updateSerializedDataSubsets()
   }
 
-  async updateSerializedDataSubsets () {
-    Object.keys(this.cache.serializedDataSubsets).forEach(async key => {
+  updateSerializedDataSubsets () {
```

Comment:
Can we add a unit test?

Reply:
There already is a unit test. They were breaking running locally on master

Reply:
Actually, i thought of a test i can add. Will update